src/asmcnc/apps/drywall_cutter_app/material_setup_popup.py

```diff
@@ -372,53 +377,51 @@ def update_graphic_position(self):
         self.float_layout.do_layout()
         self.cutter_layout.do_layout()
 
-    def update_text(self, instance, value):
+    def update_text(self):
         cutter_max_depth_total = self.dwt_config.active_cutter.max_depth_total
-        # Calculate total cut depth and handle inputs
-        if instance == self.material_thickness or instance == self.bottom_offset:
```

Comment:
I just removed this if statement and moved everything back one tab, not sure why it looks like I re-wrote the whole function